Pickers/SWActionSheet.m

```diff
@@ -208,13 +208,6 @@ - (BOOL)prefersStatusBarHidden {
 	return [UIApplication sharedApplication].statusBarHidden;
 }
 
-- (BOOL)shouldAutorotateToInterfaceOrientation:(UIInterfaceOrientation)interfaceOrientation
```

Comment:
Are you sure, that we can delete this check? I added it to fix something (shame of me, didn't put comment there).. But still.